source/benchmarking/benchmarking.rst

```diff
@@ -489,10 +489,10 @@ the database and reading a response.
 Dataset:  n/a
 
 Dataset size: While there is no external dataset, for score calculation
-purposes use 160,000 bytes (10,000 x the size of a BSON {ismaster:true}
+purposes use 130,000 bytes (10,000 x the size of a BSON {hello:true}
```

Comment:
Oh interesting, no change needed, but I wonder if this will impact these benchmarks.

Reply:
BSON `{ismaster:true}` will be serialized on the wire as:
payload size as int32
single boolean element (\x08) with value true (\x01): \x08 ismaster (8 bytes + \x00) \x01
terminal null: \x00
Total Size: 16 bytes

BSON `{hello:true}` will be serialized on the wire as:
payload size as int32
single boolean element (\x08) with value true (\x01): \x08 hello (5 bytes + \x00) \x01
terminal null: \x00
Total Size: 13 bytes

So yes, the data size will therefore be changed for this test case, which could affect the results. We can either decide to accept that `{ismaster:true}` test results can't be compared directly to `{hello:true}` results or we can artificially pad out `{hello:true}` by adding another 3 bytes such as `{a:null}` (`\x0a \x61 \x00`). My preference is to acknowledge that the test results aren't directly comparable rather than artificially padding the `hello` test to match the previous size. I suppose we could also change the number of iterations, but that would likely affect the results more than the difference in BSON document size.

Reply:
Hah, I hadn't thought about that artificial padding. I agree with your preference of keeping it as is.